backend/de.metas.acct.base/src/main/sql/postgresql/system/43-de.metas.acct/5584590_sys_gh10928_added missing_olcand_simplify_C_OLCand_Processor_WhereClause.sql

```diff
@@ -0,0 +1,7 @@
+-- add filtering for IsGroupingError
```

Comment:
pleaase rename to something without a space in the filename

Reply:
renamed.